tests/test_middleware_interface.py

```diff
@@ -224,6 +225,11 @@ def test_init(self):
         self.assertEqual(self.interface.rawIngestTask.config.failFast, True)
         self.assertEqual(self.interface.rawIngestTask.config.transfer, "copy")
 
+    def test_compute_region(self):
+        """Test preload region computation."""
+        region = self.interface._compute_region()
+        self.assertTrue(isinstance(region, lsst.sphgeom.Region))
```

Comment:
I thought of that, but was not sure how to get the "truth" that is not from the exactly same code as in `_compute_region()`.   Do you mean a comparison with a directly instantiated polygon?

Reply:
I was thinking of using the visit-detector records, with matched padding. That should give similar results, shouldn't it?

Reply:
I'm taking too long to make it work, so I decided to file a new ticket  [DM-47460](https://rubinobs.atlassian.net/browse/DM-47460) for it instead of delaying this PR. 

I moved this test to `MiddlewareInterfaceWriteableTest` where visit-detector records are available (after ingesting raws and defining visits).